### ISSUE TITLE
Shouldn't train set be excluded from unlabeled set?

### ISSUE THREAD

**Issue by PramuPerera** [jik0730/VAT-pytorch#1]
In data_loader.py, when finding instances with no labels, you have used following command:

unlabels_set = list(set(range(len(train_dataset))) - set(val_set))

shouldn't you consider indexes in train set here as well?

unlabels_set = list(set(range(len(train_dataset))) - set(val_set)-set(labels_set))

**Comment by jik0730** [comment]
Thanks for your interest on my code and sorry for the late response.
I followed experiment setting in the paper and they used unlabeled dataset to include labeled dataset.
Please let me know if I'm wrong.
